src/index.html

```diff
@@ -6,8 +6,38 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product_card" data-qa="card">
+      <img
+        class="product_photo"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27">
+
+      <div class="product_name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+      <div class="product_code">Product code: 195434</div>
```

Comment:
```suggestion
      <p class="product_code">Product code: 195434</p>
```

Reply:
I didn't change it because then everything falls apart

Reply:
add `margin: 0` for the `p` it should help

Reply:
It helped :)